service.games.discord.richpresence/addon.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon id="service.games.discord.richpresence"
+        name="Discord Rich Presence for RetroPlayer"
+        version="1.0.0"
+        provider-name="NikosSiak">
+    <requires>
+        <import addon="xbmc.python" version="3.0.1"/>
+    </requires>
+    <extension point="xbmc.service" library="default.py" />
+    <extension point="xbmc.addon.metadata">
+        <platform>all</platform>
+        <source>https://github.com/NikosSiak/service.games.discord.richpresence</source>
```

Comment:
Missing description and summary (en_GB), missing license (spdx identifier). See example:  https://github.com/xbmc/xbmc/blob/master/addons/service.xbmc.versioncheck/addon.xml